cmd/kubeadm/app/apis/kubeadm/v1beta3/doc.go

```diff
@@ -181,12 +166,15 @@ limitations under the License.
 // 	    value: "master"
 // 	    effect: "NoSchedule"
 // 	  kubeletExtraArgs:
-// 	    v: 4
+// 	    cgroup-driver: "cgroupfs"
```

Comment:
```suggestion
// 	    v: 4
```
given that cgroup-driver: "cgroupfs" is not the default anymore (I remember a PR changing this recently)